transmit-lib/src/cmr/transmit/kms.clj

```diff
@@ -61,11 +61,12 @@
   "Allow for any KMS resource URL to be overriden by the system envirnmental
    variables. The ENV is assumed to contain JSON"
   ;; assume the value comes from an ENV
-  ([] (config/kms-scheme-override-json))
+  ([] (scheme-overrides (config/kms-scheme-override-json)))
 
   ;; parse and return a map
   ([json-string] (json/parse-string json-string true)))
```

Comment:
I was going to critique this multi-arity format, but clojure style guide says your format here is good:
```
;; good for multi-arity functions
(defn foo
  ([x] (bar x))
  ([x y]
   (if (predicate? x)
     (bar x)
     (baz x))))
     
```